src/Logging.cpp

```diff
@@ -29,7 +29,8 @@ qmlMessageHandler(QtMsgType type, const QMessageLogContext &context, const QStri
         if (msg.endsWith(
               "QML Binding: Not restoring previous value because restoreMode has not been set.This "
               "behavior is deprecated.In Qt < 6.0 the default is Binding.RestoreBinding.In Qt >= "
-              "6.0 the default is Binding.RestoreBindingOrValue."))
+              "6.0 the default is Binding.RestoreBindingOrValue.") ||
+            msg.endsWith("Both point size and pixel size set. Using pixel size."))
```

Comment:
shouldn't this be an indication that we should be fixing things? seems like it could be a source of display artifacts on scaled displays.

Reply:
I should probably add a comment for that. The default style has the point size set. If you use pixel size anywhere, you get that warning, which is useless, since sometimes you need the pixel size to match the text to the size of the outer element for example. This is done in the avatar and without that you get one warning for every Avatar displayed, which is stupid!

Reply:
that would be a solid comment.